composer.json

```diff
@@ -28,7 +28,7 @@
         "behat/mink-extension": "^2.1",
         "silverstripe/mink-facebook-web-driver": "^1",
         "symfony/dom-crawler": "^3",
-        "silverstripe/testsession": "^2.1",
```

Comment:
I think we don't have to manually bump this one, as the caret should imply `>=2.1 <3`
https://getcomposer.org/doc/articles/versions.md#caret-version-range-

Reply:
oh, actually, yeah, it now requires 2.2...

Reply:
`waitForPendingRequests` does not exist in  "silverstripe/testsession": "^2.1". So behat extension won't work if you try to run it with  testsession 2.1.